client/components/config/config-directive.html

```diff
@@ -50,6 +50,35 @@
     </div>
   </div>
 
+  <div class="pk-sidebar-group-title">Security</div>
+
+  <div class="pk-sidebar-item">
+    <div class="pk-sidebar-item-label">
+      <input type="checkbox"
+             class="widget-columns-show-date"
+             ng-model="ngModel.restrict_view_to_admin">
+      <label for="restrict_view_to_admin">Restrict dashboard VIEW to Administrators</label>
```

Comment:
See my comment on your slides, I think it would be clearer to invert the checkmarks and show them as granting permissions for an "Others" or "Public" group.